sections/java.zsh

```diff
@@ -19,7 +19,7 @@ SPACESHIP_JAVA_COLOR="${SPACESHIP_JAVA_COLOR="cyan"}"
 spaceship_java() {
   [[ $SPACESHIP_JAVA_SHOW == false ]] && return
 
-  local is_java_project="$(spaceship::upsearch pom.xml build.gradle settings.gradle build.xml)"
+  local is_java_project="$(spaceship::upsearch pom.xml build.gradle build.gradle.kts settings.gradle settings.gradle.kts build.xml)"
```

Comment:
Can we use `build.gradle*` glob, basically matching every optional extension for `build.gradle` file? Or should we specify `.kts`?

I am not very familiar with Java ecosystem. Can these files have other optional extensions?

Reply:
You are right, we can use `build.gradle*` I'll refactor changes for future, however in gradle ecosystem currently be two cases: groovy: `build.gradle` and kotlin: `build.gradle.kts`

Reply:
I updated the PR changes, thanks a lot for review!